### DISCUSSION TITLE
podman machine networking

### DISCUSSION THREAD

**Discussion by abiosoft** [containers/podman#11085] [Q&A]
Firstly, I would like to applaud the efforts on the project. I was impressed with how almost effortless `podman machine` worked for me on macOS (despite being experimental).

However, I have been trying to see if there is a way to assign an ip address to the vm. Basically I need to interact with containers running in the vm from the host.

At the moment I am manually forwarding the ports with ssh (which is bearable for now) but I would like to know if there are any plans in this regard. I would be happy to help with testing.

Thanks.

**Comment by mheon** [comment]
@baude has been working on this
